Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -33,6 +33,22 @@ param (
     [string]$MeetingID
 )
 
+# ===================================================================================================
+# Auto update script
+# ===================================================================================================
+
+$BuildVersion = ""
+
+. $PSScriptRoot\..\Shared\ScriptUpdateFunctions\Test-ScriptVersion.ps1
+
+if (Test-ScriptVersion -AutoUpdate) {
```

Comment:
This will prompt the user for confirmation. If you want to do the update without asking for confirmation, you should add a `-Confirm:$false` here.